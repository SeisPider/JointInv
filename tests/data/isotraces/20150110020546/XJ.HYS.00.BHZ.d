SURF96 R U X -1 60.00000 3.77928 0.14799
SURF96 R U X -1 55.00000 3.67200 0.12806
SURF96 R U X -1 50.00000 3.58670 0.11108
SURF96 R U X -1 48.00000 3.56871 0.10557
SURF96 R U X -1 46.00000 3.56766 0.10111
SURF96 R U X -1 44.00000 3.56418 0.09652
SURF96 R U X -1 42.00000 3.53761 0.09077
SURF96 R U X -1 40.00000 3.48308 0.08380
SURF96 R U X -1 38.00000 3.43364 0.07737
SURF96 R U X -1 36.00000 3.41346 0.07244
SURF96 R U X -1 34.00000 3.39986 0.06787
SURF96 R U X -1 32.00000 3.36535 0.06258
SURF96 R U X -1 30.00000 3.33243 0.05753
SURF96 R U X -1 30.00000 3.65436 0.06918
SURF96 R U X -1 29.00000 3.31838 0.05515
SURF96 R U X -1 28.00000 3.30071 0.05268
SURF96 R U X -1 27.00000 3.28120 0.05020
SURF96 R U X -1 26.00000 3.26509 0.04787
SURF96 R U X -1 26.00000 3.46598 0.05394
SURF96 R U X -1 25.00000 3.25213 0.04566
