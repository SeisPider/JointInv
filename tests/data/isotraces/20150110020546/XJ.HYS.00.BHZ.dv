MFT96 R U T -1 60.235    3.7461    1.0000    5790.8     0.2136E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 59.362    3.7355    1.0000    5790.8     0.2313E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 58.514    3.7251    1.0000    5790.8     0.2495E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 57.690    3.7149    1.0000    5790.8     0.2684E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 56.889    3.7049    1.0000    5790.8     0.2879E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 56.110    3.6951    1.0000    5790.8     0.3079E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 55.351    3.6855    1.0000    5790.8     0.3284E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 54.613    3.6761    1.0000    5790.8     0.3494E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 53.895    3.6669    1.0000    5790.8     0.3708E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 53.195    3.6578    1.0000    5790.8     0.3925E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 52.513    3.6490    1.0000    5790.8     0.4146E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 51.848    3.6403    1.0000    5790.8     0.4370E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 51.200    3.6318    1.0000    5790.8     0.4597E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 50.568    3.6234    1.0000    5790.8     0.4826E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 49.951    3.6153    1.0000    5790.8     0.5056E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 49.349    3.6072    1.0000    5790.8     0.5288E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.762    3.5994    1.0000    5790.8     0.5520E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.188    3.5916    1.0000    5790.8     0.5753E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 47.628    3.5841    1.0000    5790.8     0.5987E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 47.080    3.5766    1.0000    5790.8     0.6220E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.545    3.5693    1.0000    5790.8     0.6453E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.022    3.5622    1.0000    5790.8     0.6685E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 45.511    3.5552    1.0000    5790.8     0.6917E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 45.011    3.5483    1.0000    5790.8     0.7147E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.522    3.5415    1.0000    5790.8     0.7376E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.043    3.5349    1.0000    5790.8     0.7604E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 43.574    3.5284    1.0000    5790.8     0.7831E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 43.116    3.5220    1.0000    5790.8     0.8056E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.667    3.5157    1.0000    5790.8     0.8279E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.227    3.5096    1.0000    5790.8     0.8502E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 41.796    3.5035    1.0000    5790.8     0.8722E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 41.374    3.4976    1.0000    5790.8     0.8942E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.960    3.4917    1.0000    5790.8     0.9161E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.554    3.4860    1.0000    5790.8     0.9379E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.157    3.4804    1.0000    5790.8     0.9597E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.767    3.4748    1.0000    5790.8     0.9814E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.385    3.4694    1.0000    5790.8     0.1003E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 39.010    3.4640    1.0000    5790.8     0.1025E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.642    3.4587    1.0000    5790.8     0.1047E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.280    3.4536    1.0000    5790.8     0.1069E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.926    3.4485    1.0000    5790.8     0.1091E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.578    3.4435    1.0000    5790.8     0.1113E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 37.236    3.4385    1.0000    5790.8     0.1135E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.901    3.4337    1.0000    5790.8     0.1158E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.571    3.4289    1.0000    5790.8     0.1181E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.248    3.4242    1.0000    5790.8     0.1203E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.930    3.4195    1.0000    5790.8     0.1226E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.617    3.4150    1.0000    5790.8     0.1249E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.310    3.4105    1.0000    5790.8     0.1273E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 35.009    3.4060    1.0000    5790.8     0.1296E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.712    3.4016    1.0000    5790.8     0.1319E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.420    3.3973    1.0000    5790.8     0.1342E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.133    3.3931    1.0000    5790.8     0.1365E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.851    3.3889    1.0000    5790.8     0.1388E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.574    3.3847    1.0000    5790.8     0.1410E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.301    3.3806    1.0000    5790.8     0.1431E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 33.032    3.3766    1.0000    5790.8     0.1453E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.768    3.3726    1.0000    5790.8     0.1473E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.508    3.3686    1.0000    5790.8     0.1492E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.252    3.3647    1.0000    5790.8     0.1510E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.000    3.3608    1.0000    5790.8     0.1527E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.752    3.3570    1.0000    5790.8     0.1543E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.508    3.3532    1.0000    5790.8     0.1557E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.267    3.3495    1.0000    5790.8     0.1569E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 31.030    3.3457    1.0000    5790.8     0.1579E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.797    3.3421    1.0000    5790.8     0.1587E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.567    3.3384    1.0000    5790.8     0.1593E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.341    3.3348    1.0000    5790.8     0.1597E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.118    3.3312    1.0000    5790.8     0.1597E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.898    3.3276    1.0000    5790.8     0.1596E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.681    3.3241    1.0000    5790.8     0.1591E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.468    3.3205    1.0000    5790.8     0.1583E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.257    3.3170    1.0000    5790.8     0.1573E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.050    3.3136    1.0000    5790.8     0.1559E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.845    3.3101    1.0000    5790.8     0.1543E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.643    3.3066    1.0000    5790.8     0.1523E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.444    3.3032    1.0000    5790.8     0.1500E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.248    3.2998    1.0000    5790.8     0.1474E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.055    3.2964    1.0000    5790.8     0.1445E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.864    3.2930    1.0000    5790.8     0.1413E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.676    3.2896    1.0000    5790.8     0.1379E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.490    3.2862    1.0000    5790.8     0.1342E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.307    3.2828    1.0000    5790.8     0.1302E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.126    3.2794    1.0000    5790.8     0.1260E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.947    3.2761    1.0000    5790.8     0.1216E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.771    3.2727    1.0000    5790.8     0.1170E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.597    3.2693    1.0000    5790.8     0.1122E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.426    3.2659    1.0000    5790.8     0.1073E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.256    3.2625    1.0000    5790.8     0.1023E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.089    3.2591    1.0000    5790.8     0.9718E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.924    3.2557    1.0000    5790.8     0.9201E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.761    3.2523    1.0000    5790.8     0.8680E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.600    3.2489    1.0000    5790.8     0.8159E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.441    3.2455    1.0000    5790.8     0.7640E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.284    3.2420    1.0000    5790.8     0.7126E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 25.129    3.2386    1.0000    5790.8     0.6618E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 60.000    3.7433    1.0000    5790.8     0.2184E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 55.000    3.6810    1.0000    5790.8     0.3384E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 50.000    3.6159    1.0000    5790.8     0.5038E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 48.000    3.5891    1.0000    5790.8     0.5832E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 46.000    3.5619    1.0000    5790.8     0.6696E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 44.000    3.5343    1.0000    5790.8     0.7625E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 42.000    3.5064    1.0000    5790.8     0.8618E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 40.000    3.4781    1.0000    5790.8     0.9684E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 38.000    3.4495    1.0000    5790.8     0.1086E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 36.000    3.4206    1.0000    5790.8     0.1221E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 34.000    3.3911    1.0000    5790.8     0.1376E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 32.000    3.3608    1.0000    5790.8     0.1527E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 30.000    3.3293    1.0000    5790.8     0.1596E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 29.000    3.3127    1.0000    5790.8     0.1555E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 28.000    3.2954    1.0000    5790.8     0.1436E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 27.000    3.2771    1.0000    5790.8     0.1229E-04   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
MFT96 R U T -1 26.000    3.2573    1.0000    5790.8     0.9438E-05   -5.655000   68.361298   42.976002   89.441002 0  1 COMMENT
