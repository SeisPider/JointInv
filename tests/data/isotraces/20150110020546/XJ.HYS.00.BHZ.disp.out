SURF96 R U T -1 60.000    3.7433    1.0000    
SURF96 R C T -1 60.000    6.5579    1.0000    
SURF96 R U T -1 55.000    3.6810    1.0000    
SURF96 R C T -1 55.000    6.1639    1.0000    
SURF96 R U T -1 50.000    3.6159    1.0000    
SURF96 R C T -1 50.000    5.8001    1.0000    
SURF96 R U T -1 48.000    3.5891    1.0000    
SURF96 R C T -1 48.000    5.6619    1.0000    
SURF96 R U T -1 46.000    3.5619    1.0000    
SURF96 R C T -1 46.000    5.5275    1.0000    
SURF96 R U T -1 44.000    3.5343    1.0000    
SURF96 R C T -1 44.000    5.3966    1.0000    
SURF96 R U T -1 42.000    3.5064    1.0000    
SURF96 R C T -1 42.000    5.2689    1.0000    
SURF96 R U T -1 40.000    3.4781    1.0000    
SURF96 R C T -1 40.000    5.1443    1.0000    
SURF96 R U T -1 38.000    3.4495    1.0000    
SURF96 R C T -1 38.000    5.0224    1.0000    
SURF96 R U T -1 36.000    3.4206    1.0000    
SURF96 R C T -1 36.000    4.9031    1.0000    
SURF96 R U T -1 34.000    3.3911    1.0000    
SURF96 R C T -1 34.000    4.7862    1.0000    
SURF96 R U T -1 32.000    3.3608    1.0000    
SURF96 R C T -1 32.000    4.6714    1.0000    
SURF96 R U T -1 30.000    3.3293    1.0000    
SURF96 R C T -1 30.000    4.5584    1.0000    
SURF96 R U T -1 29.000    3.3127    1.0000    
SURF96 R C T -1 29.000    4.5025    1.0000    
SURF96 R U T -1 28.000    3.2954    1.0000    
SURF96 R C T -1 28.000    4.4469    1.0000    
SURF96 R U T -1 27.000    3.2771    1.0000    
SURF96 R C T -1 27.000    4.3915    1.0000    
SURF96 R U T -1 26.000    3.2573    1.0000    
SURF96 R C T -1 26.000    4.3362    1.0000    
